### ISSUE TITLE
faceted search queries

### ISSUE THREAD

**Issue by mdissel** [kreeben/resin#26]
Can we expect faceting features like in solr?

**Comment by kreeben** [comment]
Yes, that shall either be solved in Resin or [Sir](https://github.com/kreeben/sir).

**Comment by sgf** [comment]
Sir's  url looks changed to https://github.com/kreeben/u-sir
